v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/SourceWriterFn.java

```diff
@@ -192,6 +192,7 @@ public void processElement(ProcessContext c) {
       skippedRecordCountMetric.inc();
       outputWithTag(c, Constants.SKIPPED_TAG, Constants.SKIPPED_TAG_MESSAGE, spannerRec);
     } else {
+      Stopwatch timer = Stopwatch.createStarted();
```

Comment:
Will this create a new `Stopwatch` object per element inside the `@ProcessElement`? 
Given a new object will be created per element, will it create extra GC/memory pressure? Looks like the `StopWatch` will be discarded after the element is processed. 

It would be great if there are more instance of this being used elsewhere or we have confidence that this will not cause issues. Let me know what you think.

Alternatively, wouldn't it just be safer to use a static method like `System.currentTimeMillis();` to get the time and subtract to find the elapsed time?